nucypher/utilities/porter/porter.py

```diff
@@ -159,14 +168,25 @@ def get_ursula_info(ursula_address) -> Porter.UrsulaInfo:
                                                   learn_on_this_thread=True,
                                                   eager=True)
 
+        timeout = 1  # self.DEFAULT_EXECUTION_TIMEOUT
```

Comment:
This was supposed to be uncommented?

```suggestion
        timeout = self.DEFAULT_EXECUTION_TIMEOUT
```

Reply:
Yes, that's left-over from testing, sorry about that.